pyccel/codegen/printing/fcode.py

```diff
@@ -2066,6 +2066,7 @@ def _print_Continue(self, expr):
 
     def _print_AugAssign(self, expr):
         new_expr = expr.to_basic_assign()
+        expr.invalidate_node()
         return self._print(new_expr)
```

Comment:
Should we not apply a similar fix to the C printer in the case where `op == '%'`?

Reply:
Done [ccfaba4](https://github.com/pyccel/pyccel/pull/1416/commits/ccfaba42ba09d6dd87ef0bfc30b8a364a93c330e)